tests/openadapt/test_scrub.py

```diff
@@ -2,11 +2,17 @@
 
 from io import BytesIO
 import os
-import warnings
 
 from PIL import Image
+import pytest
+import spacy
 
-from openadapt import config, scrub
+from openadapt import config
+
+if not spacy.util.is_package(config.SPACY_MODEL_NAME):
+    pytestmark = pytest.mark.skip(reason="SpaCy model not installed.")
```

Comment:
This does not appear to be used anywhere, is that intentional?

Reply:
It is used when running pytest.

Basically, when the user does not have the Spacy Model installed then all of the tests in the file will be skipped.

Reply:
If the model is installed then, it will import scrub and continue to run the tests